src/components/form/Obs.jsx

```diff
@@ -13,9 +13,10 @@ import withFormContext from './withFormContext';
 const Obs = (props) => {
 
   const conceptAnswerDisplay = (value, conceptAnswers) => {
+
     if (value) {
       const matchingAnswer = conceptAnswers.find(ans => ans.uuid === value);
-      return matchingAnswer ? matchingAnswer.name : null;
+      return matchingAnswer ? matchingAnswer.name.display : null;
```

Comment:
an object was being returned here instead of display value

Reply:
https://github.com/openmrs/openmrs-react-components/pull/38/files#diff-ee1fd0dbf3a469c6fa09bd65cb25a3c5R61
```
<span>{conceptAnswerDisplay(props.value, props.conceptAnswers)}</span>
```
see line